qaframework-bdd-tests/src/test/resources/features/refapp-2.x/14-clinicalVisit/clinicalVisit.feature

```diff
@@ -47,6 +47,12 @@ Feature: Clinical Visit Management
     And a user attaches patient supporting file
     And a user clicks the upload file button
     Then the system uploads the file
+    # User story: Use of clear forms button while attaching document
+    When a user clicks on Attachments link from patient visits dashboard
+    Then the system loads Attachments page
```

Comment:
You mean these workflows were missing ?

Reply:
All firefox is failing.

Reply:
It is pointing to[ add user roles ](https://github.com/openmrs/openmrs-contrib-qaframework/runs/6926994434?check_suite_focus=true#step:7:15840). There's an element that is causing the instability.